src/sentry/integrations/jira_server/utils/api.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, Any, Mapping
+
+from sentry.integrations.utils import sync_group_assignee_inbound
+from sentry.shared_integrations.exceptions import IntegrationError
+
+from ..client import JiraServerClient
+
+if TYPE_CHECKING:
+    from sentry.models import Identity, Integration, OrganizationIntegration
+
+logger = logging.getLogger(__name__)
+
+
+def _get_client(integration: Integration) -> JiraServerClient:
```

Comment:
Another file that's mostly a copy/paste from Jira Cloud's version. The only difference here is this function `_get_client` wherein I look up the org integration first so that `JiraServerClient` can receive the `default_identity.data`.

Reply:
Is there a reason why `jira_server/integration.py`'s `get_client()` method can't be used? They kinda look like they're doing the same thing, but I'm not sure if the `Integration` param is the same as `JiraServerIntegration`

Reply:
They are basically doing the same thing but I don't have access to the `self.default_identity` as in `integration.py` so I rewrote it to basically do the same thing by looking up the org integration.